instrumentation/pg/lib/opentelemetry/instrumentation/pg/patches/connection.rb

```diff
@@ -104,6 +106,8 @@ def span_attrs(kind, *args)
 
             attrs = { 'db.operation' => validated_operation(operation), 'db.postgresql.prepared_statement_name' => statement_name }
             attrs['db.statement'] = sql unless config[:db_statement] == :omit
+            collection_name = collection_name(text)
+            attrs['db.collection.name'] = collection_name unless collection_name.nil?
```

Comment:
Same here. If `collection_name` is only used to set that value, you might have it recommended to write this as: 

```ruby
attrs['db.collection.name'] = collection_name(text)
```

Also, the key/value pairs with `nil` values will get removed on line 112 with `attrs.compact!`, so it's okay to set the key even if the return value is `nil`.

Reply:
Great feedback! I didn't even realize `compact!` removes nil values.